Source/misdat.h

```diff
@@ -9,6 +9,12 @@
 
 namespace devilution {
 
+enum mienemy_type : uint8_t {

```

Comment:
Can't we use `enum class` for new enums?

Reply:
Yeah we should

One big advantage of `enum class` other than scoping is that it's not implicitly convertible to int (requires a `static_cast`)

Reply:
sure, for new ones 🤷